prac_06/guitars.py

```diff
@@ -0,0 +1,30 @@
+"""Prac06 guitars.py
+Estimated Time: 30min
+Actual Time: 41min
+"""
+
+from prac_06.guitar import Guitar
+
+print("My Guitars!")
+name = input("Name: ")
+guitars = []
+while name != "":
```

Comment:
The implementation here is good, especially with how you handle the exiting of the function. But is the error checking required? Is it explicitly stated in the requirements?

Reply:
Good point; I had assumed that it was good practice to error check input.